sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -643,9 +665,23 @@ def _start_upload(self):
     #
     # The uploader by default transfers data in chunks of 1024 * 1024 bytes at
     # a time, buffering writes until that size is reached.
+
+    # Create a request count metric
+    resource = resource_identifiers.GoogleCloudStorage(self._bucket)
+    labels = {
+        monitoring_infos.SERVICE_LABEL: 'Storage',
```

Comment:
Was the project not available on the API response after the first call? When you ran it on a pipeline were you able to see it populated on the API responses with logging?

Reply:
I debugged it and also logged it but it seems that the `acl` is empty, so I couldn't get the project.
`
<Object
 acl: []
 bucket: 'gcsio-metrics-test'
 generation: 1
 name: 'dummy_mode_file'
 size: 5242980>
`